hackvote/config/routes.rb

```diff
@@ -1,4 +1,13 @@
 Rails.application.routes.draw do
+  root        'events#new'
+
+  resources :events do
+    post 'close', on: :member
+    resources :projects do
+      post 'vote', on: :member
+    end
+  end
```

Comment:
If not all HTTP methods on a route are supported, the supported ones should be specified with `:only` or `:except`, like so:

```ruby
resources :photos, only: [:index, :show]
```

```ruby
resources :photos, except: :destroy
```